### ISSUE TITLE
can't send letters after plugin install

### ISSUE THREAD

**Issue by pikachurus** [toteph42/identy_switch#9]
roundcube 1.6.7, php-fpm 8.1, nginx 1.25, mysql 8.0.37.
just installed plugin (and migrated from ident_switch), activated in config.inc.php and then i can't send any letter, with this message in errors.log:

[14-Jun-2024 23:16:24 +0300]: <45nml2sm> PHP Error: php_network_getaddresses: getaddrinfo for %n failed: Name or service not known(POST /?_task=mail&_unlock=loading1718396184503&_framed=1&_action=send)
[14-Jun-2024 23:16:24 +0300]: <45nml2sm> PHP Error: Failed to connect socket: php_network_getaddresses: getaddrinfo for %n failed: Name or service not known (POST /?_task=mail&_unlock=loading1718396184503&_framed=1&_action=send)
[14-Jun-2024 23:16:24 +0300]: <45nml2sm> SMTP Error: Connection failed:  (Code: -1) in /var/www/roundcubemail/program/lib/Roundcube/rcube.php on line 1794 (POST /?_task=mail&_unlock=loading1718396184503&_framed=1&_action=send)

but plugin itself works - added some profiles, change between them.

with old ident_switch everything's fine

**Comment by toteph42** [comment]
Hi,
I'm on vacation until end of month. I'll then come back on your issue. I suggest in meatime you should check different SMTP setting. You could for example trace your normal RC setup and see what RC is using as host. I think it has to do with your setting. 

Am 14. Juni 2024 22:55:55 MESZ schrieb pikachurus ***@***.***>:

**Comment by CodeShakingSheep** [comment]
I run into the same issue. Seems to be a problem related so SSL.

```
[24-Jun-2024 03:30:04 UTC] PHP Warning:  stream_socket_client(): SSL operation failed with code 1. OpenSSL Error messages:
error:1408F10B:SSL routines:ssl3_get_record:wrong version number in /var/www/roundcube/vendor/pear/net_socket/Net/Socket.php on line 159
[24-Jun-2024 03:30:04 UTC] PHP Warning:  stream_socket_client(): Failed to enable crypto in /var/www/roundcube/vendor/pear/net_socket/Net/Socket.php on line 159
[24-Jun-2024 03:30:04 UTC] PHP Warning:  stream_socket_client(): Unable to connect to ssl://mail.<REDACTED_MAILSERVER>.com:587 (Unknown error) in /var/www/roundcube/vendor/pear/net_socket/Net/Socket.php on line 159
[24-Jun-2024 03:30:04 +0000]: <hqfc1cv5> PHP Error: Failed to connect socket:  (POST /?_task=mail&_unlock=loading1719199802861&_framed=1&_action=send)
[24-Jun-2024 03:30:04 +0000]: <hqfc1cv5> SMTP Error: Connection failed:  (Code: -1) in /var/www/roundcube/program/lib/Roundcube/rcube.php on line 1794 (POST /?_task=mail&_unlock=loading1719199802861&_framed=1&_action=send)
```

**Comment by toteph42** [comment]
@CodeShakingSheep Thank you for your suggestion. Will be implemented in 1.0.18
